manifests/e/Ecosia/EcosiaBrowser/123.0.6312.21/Ecosia.EcosiaBrowser.locale.en-US.yaml

```diff
@@ -0,0 +1,33 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS1.CRLF.7-4-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Ecosia.EcosiaBrowser
+PackageVersion: 123.0.6312.21
+PackageLocale: en-US
+Publisher: The Ecosia Browser Authors
+PublisherUrl: https://www.ecosia.org/
+PublisherSupportUrl: https://ecosia.helpscoutdocs.com/
+PrivacyUrl: https://www.ecosia.org/privacy
+Author: Ecosia GmbH
+PackageName: Ecosia Browser
+PackageUrl: https://www.ecosia.org/browser
+License: Freeware
+LicenseUrl: https://www.ecosia.org/terms-of-service
```

Comment:
This might be due to a network block of data centers, to prevent automated downloads.

```suggestion
# PublisherUrl: 
PublisherSupportUrl: https://ecosia.helpscoutdocs.com/
# PrivacyUrl: 
Author: Ecosia GmbH
PackageName: Ecosia Browser
# PackageUrl: 
License: Freeware
# LicenseUrl: 
```